src/results/thermal_storage.jl

```diff
@@ -27,16 +27,19 @@
 # OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 # *********************************************************************************
+"""
+`HotStorage` results:
+- `size_gal` Optimal TES capacity, by volume [gal]
+- `year_one_soc_series_pct` Vector of normalized (0-1) state of charge values over the first year [-]
+- `year_one_to_load_series_mmbtu_per_hour` Vector of power used to meet load over the first year [MMBTU/hr]
+"""
+
 """
     add_hot_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
 
-Adds the Storage results to the dictionary passed back from `run_reopt` using the solved model `m` and the `REoptInputs` for node `_n`.
+Adds the `HotStorage` results to the dictionary passed back from `run_reopt` using the solved model `m` and the `REoptInputs` for node `_n`.
```

Comment:
@NLaws  or @zolanaj will this result key be `HotStorage` or just `Storage` (or something else?)

Reply:
> 

Update: I've added doc strings for all results keys.

Reply:
Looks like `HotThermalStorage` (based on https://github.com/NREL/REopt.jl/blob/4b0fb7f6556b2b6e9a9a7e8fa65398096fb6610f/test/scenarios/thermal_storage.json#L35)

Reply:
updated in https://github.com/NREL/REopt.jl/pull/82/commits/536a4a30acea4fccda4ebfc6a059fe563e55913e